examples/t1.qasm

```diff
@@ -15,26 +15,26 @@ kernel tabulate(int[32], int[32], int[32]);
 
 bit c0, c1;
 
-// define a gate calibration for an X gate on qubit 1
-defcal "openpulse" x %0 {
-   play drive(%0), gaussian(100, 30, 5);
+// define a gate calibration for an X gate on any qubit
+defcal "openpulse" x $q {
```

Comment:
While we are at it, we should remove the `"openpulse"` from the `defcal`, which has been dropped from the spec.

Reply:
How does the user know then what pulse grammar to use? It still appears at the bottom of https://qiskit.github.io/openqasm/language/pulses.html as well?

Reply:
With a `defcalgrammar` statement. I should fix that section you point out as well.

Reply:
Ok--I didn't realize that was the sole option now. Will update.

Reply:
@blakejohnson I fixed that issue as well in this PR